bot-components/GitHub_subscriptions.ml

```diff
@@ -26,6 +26,13 @@ type pull_request_action =
 type 'a pull_request_info =
   {issue: 'a; base: commit_info; head: commit_info; merged: bool}
 
+type 'a pull_request_info_commit_message =
+  { issue: 'a
+  ; base: commit_info
+  ; head: commit_info
+  ; merged: bool
+  ; last_commit_message: string }
```

Comment:
To avoid inflation of data types, an alternative could have been to make the `last_commit_message` field an option.